io/shared/src/main/scala/org/scalajs/io/MemFiles.scala

```diff
@@ -65,6 +67,9 @@ class MemVirtualBinaryFile(p: String) extends MemVirtualFile(p)
     content = v
     this
   }
+
+  final def withUTF8String(v: String): this.type =
```

Comment:
By your own reasoning in `fromStringUTF8`, this one should be called `withStringUTF8`, shouldn't it?

Reply:
Yes